Jenkinsfile.deploy

```
@@ -144,7 +147,7 @@ pipeline {
         stage('Check kogito-examples with events') {
             steps {
                 script {
-                    sh 'cp -r kogito-examples kogito-examples-events'
+                    sh "cp -r ${getRepoName()} ${getRepoName()}-events"
                     getMavenCommand()
```

I guess this is a bug coming from previous code ...
```suggestion
                    getMavenCommand("${getRepoName()}-events")
```